libtransmission/rpcimpl.cc

```diff
@@ -134,7 +134,7 @@ auto getTorrents(tr_session* session, tr_variant* args)
                 std::begin(by_id),
```

Comment:
```suggestion
                std::begin(by_id)+1,
```
You could skip the first entry, as it'll always be `nullptr`

Reply:
It doesn't make any difference, tbh, as it won't be copied.